src/v3/wrap.ts

```diff
@@ -13,16 +13,34 @@ const deepMap = (value: any, path: string): Salt[] => {
   if (Array.isArray(value)) {
     return value.flatMap((v, index) => deepMap(v, `${path}[${index}]`));
   }
-  if (typeof value === "object") {
+  if (typeof value === "object" && value) {
```

Comment:
let's add a comment on this one to explain the && value

Reply:
added a comment to explain the reasoning